internal/auth0/organization/resource_connection.go

```diff
@@ -60,13 +76,18 @@ func createOrganizationConnection(ctx context.Context, data *schema.ResourceData
 	api := meta.(*config.Config).GetAPI()
 
 	organizationID := data.Get("organization_id").(string)
-
 	connectionID := data.Get("connection_id").(string)
 	assignMembershipOnLogin := data.Get("assign_membership_on_login").(bool)
 
+	// We need to keep these null if they don't appear in the schema.
+	isSignupEnabled := value.Bool(data.GetRawConfig().GetAttr("is_signup_enabled"))
+	showAsButton := value.Bool(data.GetRawConfig().GetAttr("show_as_button"))
+
 	organizationConnection := &management.OrganizationConnection{
```

Comment:
Given this is becoming more chunky, can we follow the same pattern of expand through out the codebase? Where we abstract this away into an `orgConn := expandOrganizationConnection(data)` func.

Reply:
see below :-)